src/components/_data/ContentSingularData.ts

```diff
@@ -1,6 +1,7 @@
 export type ContentSingularData = {
-    plainText_content?: string
-    exampleImage_imageLink?: string
-    exampleImage_percentageSize?: number
-    [idx: string]: any
-}
\ No newline at end of file
+  plainText_content?: string;
+  exampleImage_imageLink?: string;
+  exampleImage_percentageSize?: number;
+  toDoList_content?: [{ text?: string; isChecked?: boolean }];
```

Comment:
`text` and `isChecked` are required by your things or they will likely crash. I would just make them required.